.github/workflows/ci.yml

```diff
@@ -0,0 +1,194 @@
+        uses: "actions/checkout@v2"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v1"
+      - name: "Cache Docker images"
+        uses: "actions/cache@v2"
+        id: "cached-docker-images"
+        with:
+          path: "/tmp/docker"
+          key: "${{ runner.os }}-docker-${{ matrix.python-version }}-${{ hashFiles('./docker/*') }}-1"
+      - name: "Load docker image"
+        run: "docker load < /tmp/docker/nautobot-dev-py${{ matrix.python-version }}.tar"
+      - name: "Show docker images"
+        run: "docker image ls"
+      - name: "Check migrations"
+        run: "poetry run invoke check-migrations"
+    needs:
+      - "build"
+  tests-postgres:
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9"]
+    env:
+      INVOKE_NAUTOBOT_PYTHON_VER: "${{ matrix.python-version }}"
+    runs-on: "ubuntu-20.04"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v1"
+      - name: "Cache Docker images"
+        uses: "actions/cache@v2"
+        id: "cached-docker-images"
+        with:
+          path: "/tmp/docker"
+          key: "${{ runner.os }}-docker-${{ matrix.python-version }}-${{ hashFiles('./docker/*') }}-1"
+      - name: "Load docker image"
+        run: "docker load < /tmp/docker/nautobot-dev-py${{ matrix.python-version }}.tar"
+      - name: "Show docker images"
+        run: "docker image ls"
+      - name: "Run unittest"
+        run: "poetry run invoke unittest --failfast --keepdb"
+      - name: "Run unittest coverage"
+        run: "poetry run invoke unittest-coverage"
+    needs:
+      - "check_migrations"
+  tests-mysql:
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9"]
+    env:
+      INVOKE_NAUTOBOT_PYTHON_VER: "${{ matrix.python-version }}"
+    runs-on: "ubuntu-20.04"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v1"
+      - name: "Cache Docker images"
+        uses: "actions/cache@v2"
+        id: "cached-docker-images"
+        with:
+          path: "/tmp/docker"
+          key: "${{ runner.os }}-docker-${{ matrix.python-version }}-${{ hashFiles('./docker/*') }}-1"
+      - name: "Load docker image"
+        run: "docker load < /tmp/docker/nautobot-dev-py${{ matrix.python-version }}.tar"
+      - name: "Show docker images"
+        run: "docker image ls"
+      - name: "Copy mysql creds"
+        run: "cp ./development/dev-mysql.env ./development/dev.env"
+      - name: "Create Docker network"
+        run: "docker network create --driver bridge nautobot_default"
+      - name: "Start mysql"
+        run: "docker run --name mysql --rm --env-file ./development/dev.env --network nautobot_default -itd mysql"
```

Comment:
Better way to do this would be with an `invoke.yml` and `override.env` - take a look at #953, we may want to backport it to `develop` if it would be useful here.

Reply:
Yup, the linked PR definitely looks better...